cron.yaml

```diff
@@ -20,12 +20,6 @@ cron:
 - description: weekly exploration search rank computation
   url: /cron/explorations/search_rank
   schedule: every wednesday 9:00
-- description: accept all stale suggestions every week
-  url: /cron/suggestions/accept_stale_suggestions
-  schedule: every thursday 9:00
-- description: notify reviewers about suggestions requiring review
-  url: /cron/suggestions/notify_reviewers
```

Comment:
This URL wasn't linked to any controller and it was throwing error on the server!

Reply:
The handler got deleted in the following PR: https://github.com/oppia/oppia/pull/7893/files#diff-44acfdd1db81ed9d8d37c573b0b12cc7L213